content/response.md

```diff
@@ -23,36 +32,39 @@ In a text detection context based on a small dataset of 229 fully annotated scen
 
 > On a similar note, it is mentioned that AUCROC may not be reliable, but consider recommending alternatives such as precision-recall curves.
 
 We appreciate the suggestion to include an alternative and have added the follwing sentence: "One alternative approach is to use the precision-recall curve rather than the receiver operating characteristic since the former is more resistant to imbalanced data [@doi:10.1145/1143844.1143874]."
 
->
 > ### Minor comments:
-> - Lines 153: those though
+>
+> - Lines 153: those though
 
 We have corrected this typo.
->
+
 > - Line 183 is missing a period
 
 We have added the missing period.
 
 > - Paragraph starting at line 271: This paragraph is cumbersome and contains jargon that is not introduced (CUDA/CuDNN). Is it implied that implementation (hardware and software) variations between platforms can hinder reproducibility? Or perhaps the implementation is reproducible, so long that random seeding is not used?
->
 
 The paragraph now reads:
 
-A specific reproducibility pitfall that is often missed in applying deep learning is the use (often by default) of non-deterministic algorithms.
-For example, the GPU acceleration libraries, such as CUDA/CuDNN, that facilitate the parallelized computing powering state-of-the-art deep learning often use algorithms by default that produce different outcomes from iteration to iteration even with the same hardware and sofrware.
-Therefore, achieving reproducibility in this context requires explicitly specifying the use of deterministic algorithms (which are typically available within deep learning libraries).
-This step is distinct from and in addition to the setting of random seeds that typically achieve reproducibility by controlling pseudorandom deterministic procedures such as shuffling and initialization [@url:https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#reproducibility].
\ No newline at end of file
+> A specific reproducibility pitfall that is often missed in applying deep learning is the use (often by default) of non-deterministic algorithms.
+> For example, the GPU acceleration libraries, such as CUDA/CuDNN, that facilitate the parallelized computing powering state-of-the-art deep learning often use algorithms by default that produce different outcomes from iteration to iteration even with the same hardware and sofrware.
+> Therefore, achieving reproducibility in this context requires explicitly specifying the use of deterministic algorithms (which are typically available within deep learning libraries).
+> This step is distinct from and in addition to the setting of random seeds that typically achieve reproducibility by controlling pseudorandom deterministic procedures such as shuffling and initialization [@url:https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#reproducibility].
+
+We hope these changes serve to improve the quality of the manuscript and satisify the concerns of the reviewers.
```

Comment:
The "serve to" is a bit redundant. Could be just 

> We hope these changes improve the quality of the manuscript and satisify the concerns of the reviewers.

Reply:
```suggestion
We hope these changes improve the quality of the manuscript and satisfy the concerns of the reviewers.
```